src/components/AddContacts.js

```diff
@@ -0,0 +1,58 @@
+
+import { Link } from 'react-router-dom';
+import React, { useState } from 'react';
+
+
+const AddContacts = (props) => {
+
+  const [img, setImg] = useState('')
+  const handleImgChange = (e) => {
+    setImg(e.target.value)
+  }
+
+  const [name, setName] = useState('')
+  const handleNameChange = (e) => {
+  setName(e.target.value)
+  }
+
+  const [email, setEmail] = useState('')
+  const handleEmailChange = (e) => {
+  setEmail(e.target.value)
+  }
+
+  const [phone, setPhone] = useState('')
+  const handlePhoneChange = (e) => {
+  setPhone(e.target.value)
+  }
+
+  const passState = () => {
+    const contact = {
+      img: {img},
+      name: {name},
+      email: {email},
+      phone: {phone}
+    }
+    props.mainState(contact)
```

Comment:
This is where you'd want to redirect back to the main page. Look up `history` from React Router.